components/dash-table/tests/selenium/test_basic_copy_paste.py

```diff
@@ -322,15 +348,75 @@ def test_tbcp010_copy_from_unselectable_cells_table(test):
     source.cell(2, 2).double_click()
     assert source.cell(2, 2).get_text() == test.get_selected_text()
 
-    # copy the source text to clipboard using CTRL+C
+    # copy the source text to clipboard using CTRL+C or COMMAND+C
     test.copy()
+    # Uncomment the following two lines if using a Mac computer
+    # with test.hold(Keys.COMMAND):
+    #     ActionChains(test.driver).send_keys("c").perform()
```

Comment:
Oh interesting, I thought I had run these tests before on my Mac but I guess not because you're right, they fail. The better way to do this would be to modify `test.paste` etc so they use `Keys.COMMAND` on Mac and `Keys.CONTROL` on others.

In fact we do exactly that in one place (which has an older pattern in other ways):

https://github.com/plotly/dash/blob/29a389245c8c44802d55ba847b6c2e54a28fe2eb/components/dash-table/tests/selenium/conftest.py#L271

We just need to do the same in copy and paste:

https://github.com/plotly/dash/blob/29a389245c8c44802d55ba847b6c2e54a28fe2eb/components/dash-table/tests/selenium/conftest.py#L520-L526

The cleanest way to do this would be to pull the `CMD =` line up to the top of the module and then use `CMD` in copy & paste.